sms_twilio.js

```diff
@@ -14,6 +14,38 @@ sms.configure = function (options) {
   sms.from = options.from;
 };
 
+var lookups = new Meteor.Collection('meteor_accounts_sms_lookups');
+
+/**
+ * Lookup phone number information through twilio lookups api.
+ * @param phone
+ */
+sms.lookup = function (phone) {
+  var lookup = lookups.findOne({ phone_number: { $regex: phone }});
```

Comment:
Maybe have a check prior making sure phone is a non empty string.

Not sure about the regex match - `1` would match `111` - any special use case you had in mind?